resources/containers/field-modal.tsx

```diff
@@ -58,59 +60,88 @@ const FieldModal: React.FC<FieldModalProps> = ({
     );
   };
 
-  const renderMultiSelect = () => (
-    <div className={styles.multiselectContainer}>
-      <div
-        className={styles.selectedOptions}
-        onClick={() => setIsDropdownOpen(!isDropdownOpen)}
-      >
-        {selectedOptions.map((option, index) => (
-          <div key={index} className={styles.optionTag}>
-            {option}
-            <button
-              onClick={(e) => {
-                e.stopPropagation();
-                handleRemoveOption(option);
-              }}
-            >
-              ×
-            </button>
-          </div>
-        ))}
-        <input
-          type="text"
-          readOnly
-          placeholder="Select options..."
-          className={styles.input}
+  const renderContent = () => {
+    if (fieldType === TYPE_OPTIONS.NUMBER) {
+      return (
+        <BasicInput
+          type="number"
+          value={selectedOptions[0] || ""}
+          onChange={(e) => setSelectedOptions([e.target.value])}
+          placeholder="Enter max value"
         />
-      </div>
-      {isDropdownOpen && (
-        <div className={styles.dropdown}>
-          {options.map((option) => (
-            <div
-              key={option}
-              className={styles.dropdownOption}
-              onClick={() => handleOptionSelect(option)}
-            >
-              {option}
+      );
+    } else if (fieldType === TYPE_OPTIONS.SELECT) {
+      return (
+        <div className={styles.multiselectContainer}>
+          <div
+            className={styles.selectedOptions}
+            onClick={() => setIsDropdownOpen(!isDropdownOpen)}
+          >
+            {selectedOptions.map((option, index) => (
```

Comment:
Requested the uses or Resources here 🔥  

And the use of reac_hook

Reply:
If you don't undertand my comnets, create a new task on the board to do next week, if this stable I can accept for now